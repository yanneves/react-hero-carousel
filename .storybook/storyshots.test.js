import configure from "@storybook/addon-storyshots";

configure();
