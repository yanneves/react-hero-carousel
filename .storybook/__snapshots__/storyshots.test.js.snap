// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Example Background Images 1`] = `
<div
  className="wrapper"
>
  <div
    className="slide active"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=1&txt=1)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=2&txt=2)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=3&txt=3)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=4&txt=4)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=5&txt=5)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots Example Custom Interval 1`] = `
<div
  className="wrapper"
>
  <div
    className="slide active"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=1&txt=1)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=2&txt=2)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=3&txt=3)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=4&txt=4)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <article
      style={
        Object {
          "backgroundImage": "url(https://placem.at/places?w=1920&h=1080&random=5&txt=5)",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "cover",
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        }
      }
    />
  </div>
</div>
`;

exports[`Storyshots Example Images 1`] = `
<div
  className="wrapper"
>
  <div
    className="slide active"
  >
    <img
      alt="Placeholder 1"
      src="https://placem.at/places?w=1920&h=1080&random=1&txt=1"
      style={
        Object {
          "height": "100%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <img
      alt="Placeholder 2"
      src="https://placem.at/places?w=1920&h=1080&random=2&txt=2"
      style={
        Object {
          "height": "100%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <img
      alt="Placeholder 3"
      src="https://placem.at/places?w=1920&h=1080&random=3&txt=3"
      style={
        Object {
          "height": "100%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <img
      alt="Placeholder 4"
      src="https://placem.at/places?w=1920&h=1080&random=4&txt=4"
      style={
        Object {
          "height": "100%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
  </div>
  <div
    className="slide"
  >
    <img
      alt="Placeholder 5"
      src="https://placem.at/places?w=1920&h=1080&random=5&txt=5"
      style={
        Object {
          "height": "100%",
          "objectFit": "cover",
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;
